lib/fog/azurerm/docs/storage.md

```diff
@@ -176,42 +166,119 @@ result = container.destroy
 puts "#{result}"
 ```
 
-### Metadata
+## Upload a local file as a blob
+```ruby
+new_blob = azure_storage_service.blobs.get('<Container name>', '<Blob name>').create(file_path: '<file path>')
```

Comment:
The collection's get method is to get a local object which is a reference to the cloud blob object, it could not exist on the server. So creating a local object can save calls to uploading a real blob, as I am going to add API like: `blob.exists?`.

However, if you feel like a `blobs.create(container_name, blob_name, file_path)` is necessary, I am happy to add this API. Thoughts?